Add CfP page (#454)
EuroPython/website: data/pages-content/cfp.md

---
title: CFP
subtitle: EuroPython Call for Proposals
---
# Call for Proposals

<div style={{textAlign: "center",  marginBottom: 10}}>
**EuroPython 2024 Call for Proposals (CFP) is Open.**

It will be running between
**Wednesday, 14th February 2024** and **Wednesday, 6th March 2024**
[Anywhere on Earth](https://en.wikipedia.org/wiki/Anywhere_on_Earth)
</div>

> 

Author: ```suggestion
<div style={{textAlign: "center",  marginBottom: 10}}>
**Submit your application in [program.europython.eu](https://program.europython.eu)**
</div>
```

> </div>
>  <ButtonLink href="https://program.europython.eu">Submit your proposal now!🐍</ButtonLink>

Author: ```suggestion
 <ButtonLink href="https://program.europython.eu">Submit your proposal now!🐍</ButtonLink>
 </div>
```

  may introduce new (more advanced) topics.

* **Advanced** - the audience is full of subject matter experts... go ahead and
  really geek out.

In previous years we’ve had many beginner and intermediate level talks. **We’d
love to encourage more advanced level contributions too.** Come geek out with
us at the conference!

## Topics ##

We welcome presentations on any topic that will be of specific or general
interest to members of the Python community. We are a broad and deep community,
so talks that may, at first, seem niche or unexpected actually make EuroPython
the cosmopolitan place that it is. **It's your conference!**

Browse through our [past
conferences](https://www.europython-society.org/europython/) and [talk
videos](https://www.youtube.com/c/EuroPythonConference) for more ideas and to
learn where there may be gaps that ought to be filled.

For ideas to contribute to the PyData track, take a look at PyData events:
[pydata.org](https://pydata.org/).

## Selection Process ##

To ensure our programme reflects our diverse community, submissions go through
three rounds of review and refinement before the final programme is announced:

> * **[Community voting]** (from March 8th, until Mrach 15th): a voting round open to

Reviewer: @cmaureir looking at this on the website, are the `[]` leftovers from an old link?

Author: yes they are :P

---
title: CFP
subtitle: EuroPython Call for Proposals
---
# Call for Proposals

<div style={{textAlign: "center",  marginBottom: 10}}>
**EuroPython 2024 Call for Proposals (CFP) is Open.**

It will be running between
**Wednesday, 14th February 2024** and **Wednesday, 6th March 2024**
[Anywhere on Earth](https://en.wikipedia.org/wiki/Anywhere_on_Earth)
</div>

---

EuroPython reflects the colourful and diverse backgrounds, cultures and
interests of our community, so you (yes, you!) should go for it: **propose

> something and represent!**

Reviewer: “and represent”... what?
“propose something and present”?
“propose something to present at EuroPython”?

Reviewer 2: It is a youthful way of encouraging people to be a symbol or to be an outstanding example of whatever they choose. It is deliberately used here as an intransitive verb.

Author: Yeah, you can use it even as an expression "Represent!" :)

No matter your level of Python or public speaking experience, EuroPython's job
is to help you bring yourself to our community so we all flourish and benefit
from each other's experience and contribution.

If you’re thinking, “but they don’t mean me”, then we especially mean **YOU**.

* If you’re from a background that isn't usually well-represented in most
  Python groups, get involved - we want to help you make a difference.

* If you’re from a background which is well-represented in most Python groups,
  get involved - we want your help making a difference.

> * If you’re worried about not being technical enough, get involved - your fresh

Reviewer: I don't think it's the person that's technical or not. It's their knowledge or the things they want to talk about.
```suggestion
* If you’re worried about not having enough technical knowledge, get involved - your fresh
```

Author: It's just an informal way of referring to the same idea, but I like your suggestion.

  perspective will be invaluable.

* If you think you’re an imposter, join us - many of the EuroPython organisers
  feel this way.

* This is a volunteer led community, so join us - you will be welcomed with
  friendship, support and compassion.

You are welcome to share your questions and ideas with our programme team at
[programme@europython.eu](mailto:programme@europython.eu).

## The Conference Format ##

> The conference will feature around 16 Tutorials/Workshops and 120 talk sessions:

Reviewer: ```suggestion
The conference will feature around 16 tutorials/workshops and 120 talk sessions:
```

> * **Tutorials/Workshops (July 8th and 9th)**: hands-on **180-mins sessions** with

Reviewer: ```suggestion
* **Tutorials/workshops (July 8th and 9th)**: hands-on **180 minute sessions** with
```

(I'd say to either use “min” on both bullet points or “minute” on both bullet points, but not one of each.)

  40-100 people

* **Talks (July 10th, until 12th)** : **30 or 45 minutes** : presentations of specific
  and general interest to the European Python community (including a dedicated

>   PyData track)

Author: Are these dates correct? The conference runs from 8th until 14th, so I expect:
* 8 - 9 July: Tutorials
* 10 - 12 July: Talks/Posters
* 13 - 14 July: Sprints

right?

Reviewer 3: yes, those are correct dates!

>   PyData track)

Author: Is this intended? I'm not a native speaker but "45 minute" certainly reads strange to me. It might be my ignorance, but besides '1 minute' I was under the impression all the other amounts included an 's'.

>   PyData track)

Reviewer: This was intended but _I_ may be wrong.
When counting minutes by themselves, you do say 1 minute, 2 minuteS, 3 minuteS, etc, but in this context I thought you dropped the s.
I can't explain why. Maybe I'm wrong!

>   PyData track)

Reviewer 2: When 'minute' is used as a compound adjective, such as in 'a 45-minute presentation,' it is singular and should have a hyphen. However, when used as a noun, it should be plural when referring to more than one minute, as in 'the presentation will last 45 minutes.' Because '30-or-45-minute talks' may appear awkward, changing it to a noun format like 'talks: 30 or 45 minutes' might improve readability. In this case, 'minute' is not modifying 'talks,' so it should be plural. This also applies to the abbreviated format 'min.'

>   PyData track)

Reviewer: Thank you so much for explaining @hypha!

You are also welcome to submit **posters** - printed in large formats. It is
a graphical way to showcase your research, project or technology. Posters are
exhibited at the conference, read at any time by participants, and can be
discussed face-to-face with their authors during the poster sessions between

> July 10th until 12th.

Reviewer: ```suggestion
July 10th and 12th.
```

  may introduce new (more advanced) topics.

* **Advanced** - the audience is full of subject matter experts... go ahead and
  really geek out.

In previous years we’ve had many beginner and intermediate level talks. **We’d
love to encourage more advanced level contributions too.** Come geek out with
us at the conference!

## Topics ##

We welcome presentations on any topic that will be of specific or general
interest to members of the Python community. We are a broad and deep community,
so talks that may, at first, seem niche or unexpected actually make EuroPython
the cosmopolitan place that it is. **It's your conference!**

Browse through our [past
conferences](https://www.europython-society.org/europython/) and [talk
videos](https://www.youtube.com/c/EuroPythonConference) for more ideas and to
learn where there may be gaps that ought to be filled.

For ideas to contribute to the PyData track, take a look at PyData events:
[pydata.org](https://pydata.org/).

## Selection Process ##

To ensure our programme reflects our diverse community, submissions go through
three rounds of review and refinement before the final programme is announced:

> * **[Community voting](/voting)** (from March 8th, until Mach 15th): is open to

Author: "Community **voting**" -> "a voting round"
sounds a bit redundant to me, but I don't have a strong opinion.

> * **[Community voting](/voting)** (from March 8th, until Mach 15th): a voting round open to

Author: I'd say we better remove this link, because it's not online
```suggestion
* **Community voting** (from March 8th, until Mach 15th): a voting round open to
```

Reviewer 2: It would probably be "March" instead of "Mach", though Mach sounds more exciting 😊

Reviewer 3: It definitely does :D Thanks!

>   all 2015-2024 EuroPython conference ticket holders, and anyone who proposes

Reviewer: ```suggestion
  all 2015-2024 EuroPython conference ticket holders and anyone who proposes
```

  a talk for the 2024 conference. This helps us judge both the uniqueness,
  niche-ness, popularity and interestingness of each proposal.

* **Panel feedback** (from March 8th, until Mach 15th): a diverse group of
  experienced reviewers refine and provide constructive feedback on submissions
  within their realm of expertise. This helps us to judge the balance, quality
  and diversity of our programme.

* **Programme team** - appointed by the EuroPython Society, these volunteers
  curate the final programme based upon the results of community voting and
  panel feedback. They also ensure we meet editorial criteria such as
  diversity, depth and breadth of topics and speakers.

## First-Time Speakers ##

As in previous years, there will be a **Speaker Mentorship Programme** for
anyone who needs help preparing their contribution. We focus on those for whom
it might be their first time contributing to our conference, especially those

> from under-represented backgrounds. To this end we will match mentees with

Reviewer: ```suggestion
from under-represented backgrounds. To this end, we will match mentees with
```

experienced and supportive mentors and run two workshops online: one close to
the CFP closing date, focusing on improving your submissions; the other close

> to the conference date, helping you improve your talk.

Reviewer: ```suggestion
to the conference date, focusing on helping you improve your talk.
```

Mentorship Programme sign-up has started on **January 23rd** and will remain
open until TBD February. More details can be found on our [Mentorship

> page](/mentorship).

Author: @thepetk Do we have a deadline for this? I hope it's after February 14th :P

## Speaker Support

We hope to meet as many speakers as possible in person. To make attendance more
attractive, **accepted speakers are eligible for a free Conference Ticket;
accepted workshop & tutorial organisers are eligible for a free Combined

> Ticket** (see details of the [ticket types](/tickets#ticket-types)). Travel and

Author: ```suggestion
Ticket**. Travel and
```
This also is not available.

accommodation expenses will not be reimbursed,  but our [Financial Aid

> programme](/finaid) is open for applications, and we encourage speakers,

Author: This would need the finaid page to be up as well, or we can replace this with  a "More information coming soon".

> especially those from under-represented backgrounds to apply.

Reviewer: ```suggestion
especially those from under-represented backgrounds, to apply.
```
